frontend/src/components/Layout/Home/Home.js

```diff
@@ -26,23 +26,13 @@ const GetStarted = () => {
         variant="body1"
         style={{ color: "#3D3D3D", fontSize: "18px", marginBottom: "50px" }}
       >
-        fAIr is an open AI-assisted mapping service developed by the
-        Humanitarian OpenStreetMap Team (HOT) that aims to improve the
-        efficiency and accuracy of mapping efforts for humanitarian purposes.
-        The service uses AI models, specifically computer vision techniques, to
-        detect objects such as buildings, roads, waterways, and trees from
-        satellite and UAV imagery. The name fAIr is derived from the following
-        terms:
+        fAIr performs mapping in the same way as human mappers using HOT's Tasking Manager. It looks at UAV imagery and produces map data that can be added to OpenStreetMap (OSM). Tests show a 100% speedup compared to manual mapping. It uses Artificial Intelligence (AI) to accomplish this.
```

Comment:
I am little bit curious of 100 % number , Is it coming from some analysis ? Just curious

Reply:
I think the 100% sppedup is coming from [this article](https://www.hotosm.org/tech-blog/hot-tech-talks-fair/) Why fAIr? section

based on the average 1000-1500 buildings mapped per working day without AI assistance and the pilot when the average mapping nearly doubled to 2500-3000.

@natrimmer, confirm? 
or do you have other sources?

Reply:
Yes that data point from @Ettrig 's suggested copy replacement does seem to come from that blog post.

Reply:
Okay thanks both